src/functionalTest/resources/features/F-204 - Get NoC Questions/S-204.4.td.json

```diff
@@ -28,8 +28,7 @@
   "expectedResponse": {
     "_extends_": "ACA_400_Response",
     "body": {
-      "message" : "Case ID has to be a valid 16-digit Luhn number",
-      "code" : "case-id-invalid",
+      "message" : "getNoticeOfChangeQuestions.caseId: Case ID has to be a valid 16-digit Luhn number",
```

Comment:
We shouldn't be removing the new `code` error response value requested by EXUI under [ACA-133](https://tools.hmcts.net/jira/browse/ACA-133).  

Hopefully if the proposed adjustment to resolve your concerns [here](https://github.com/hmcts/aac-manage-case-assignment/pull/227/files#r573794724) works this code value should return.

Reply:
OK. tbh XUI is interested in business specific NoC error codes only not structural validation like caseId etc. (not sure why ACA-133 has all). Currently caseId validation error code is applied to few apis only (nothing in noc-approval, apply-noc-decision, case-assignments apis). Because of this we are maintaining two versions of ErrorCode classes  plus slightly complicated code in the RestExceptionHandler.  
Anyway fixed above issue by handling ConstraintViolationException

Reply:
Thanks.  Having fought to keep using the error constants in our test routines whenever possible rather than repeating them: I agree it does feel disappointing that we now have the same validation messages repeated in three location in the main code, including twice in the same class.